src/actions/graph.js

```diff
@@ -93,10 +80,12 @@ const chooseLayout = (instance, state) =>
 const layoutWorker = preventWorkerQueues(worker, chooseLayout);
 
 /**
- * Formula to determine if the pipeline is chonky
+ * Formula to determine if the pipeline is large, which is simply the sum of
+ * both the amount of nodes and edges within the graph. Edges are given a
+ * strongger weight of 1.5 with it being more computational heavy to render.
  */
-const isChonky = (chonkyNodeAmount, nodesNo, edgesNo) => {
-  return nodesNo + 1.5 * edgesNo > chonkyNodeAmount ? true : false;
+const isLarge = (largeNodeCountThreshold, nodeCount, edgeCount) => {
+  return nodeCount + 1.5 * edgeCount > largeNodeCountThreshold ? true : false;
```

Comment:
I didn't notice this earlier, but since this threshold is in terms of both `nodeCount` and `edgeCount` it could be better to go with `largeGraphThreshold` actually?

Reply:
updated with this new naming in latest commit